library/std/src/process.rs

```diff
@@ -557,6 +557,9 @@ impl Command {
     ///
     /// [`args`]: Command::args
     ///
+    /// Note that the argument is passed to the program directly as is, so you shouldn't wrap it in quotes
+    /// or escape special characters the same way you would do that when running the program from terminal.
```

Comment:
Maybe something like this?

```suggestion
    /// Note that the argument is not passed through a shell, but given
    /// literally to the program. This means that shell syntax like quotes,
    /// escaped characters, word splitting, glob patterns, substitution, etc.
    /// have no effect.
```

This is the same reason why `.arg()` cannot be used for multiple arguments at once, as explained above in the same doc comment:

> `/// Only one argument can be passed per use.`

So, these two notes could be combined. But it's fine to leave that as is. That could be a separate change.